config/database.js

```diff
@@ -0,0 +1,33 @@
+import { env } from '../helpers/utils';
+
+/**
+ * Get the default configuration setup
+ *
+ * @param {Object} options - config options
+ * @returns {Object} config object
+ */
+const getConfig = options => ({
+  username: env('DB_USERNAME', 'postgres'),
+  password: env('DB_PASSWORD', ''),
+  database: env('DB_DATABASE', ''),
+  host: env('DB_HOST', '127.0.0.1'),
+  dialect: env('DB_CONNECTION', 'postgres'),
+  ...options,
+});
+
+/**
+ * Sequelize config options
+ */
+const options = {
+  logging: env('DB_LOGGING', false), // disable query logging
+  operatorsAliases: false, // Optional: disable deprecation warning
+};
+
+export default {
+  development: getConfig(options),
+  test: getConfig(options),
```

Comment:
Can you please setup an external DB for testing to not break Travis CI/CD

Reply:
Will work on it.

Reply:
Issue resolved 👍